interfaces/Portalicious/src/app/utils/form-validation.ts

```diff
@@ -0,0 +1,16 @@
+import { FormGroup } from '@angular/forms';
+
+export function generateFieldErrors<T extends FormGroup>(
```

Comment:
I'm not sure how confusing this makes the readability of the code in the components.

I added this mainly as a wrapper because I found myself checking `formGroup.controls[controlName].touched` everywhere an error on a field was shown. This way, that check isn't necessary anymore. But maybe it would be better/simpler to just be explicit and duplicate an `if (formGroup.controls[controlName].touched)` everywhere.